---
layout: archive
title: "Technical Resume"
permalink: /technical-resume/
author_profile: true
redirect_from:
  - /technical-resume
---

See PDF version [here](/files/AiswaryaPrasad_industry_resume.pdf).

<!-- {% include base_path %}

> PhD candidate in Quantitative Biology with expertise in microbiome research, bioinformatics, and hands-on lab experience. Strong focus on microbiome ecology, metagenomics, and diverse next-generation sequencing (NGS) approaches. Proven track record in leading multi-disciplinary research projects and international collaboration. Seeking opportunities to leverage my skills in a challenging innovation-focused role.

Education
======

**PhD in Quantitative Biology**
_University of Lausanne, Switzerland | AUG 2020 - AUG 2025 (planned)_
Thesis title: “Ecology and evolution of the gut microbiome in honeybees”
 + Awarded the Faculty of Biology and Medicine (FBM) PhD Fellowship and Mathilde Agassiz Scholarship (awarded to top-placed FBM fellow).
 + Presented research as posters at 3 prominent international conferences; invited as a speaker at 2. Co-authored 4 publications and performed the peer-review of 2 papers.
 + 4 years of Teaching experience as TA for the Master-level “bioinformatics through research” course; developed an R markdown template to improve and standardize course material as interactive HTML documents. Supervised semester-long student projects presented at the course.
 + Engaged as a student representative on the department Faculty Hiring Committee and the organizing committee for the annual department scientific retreat.

**B.S. and M.S. in Biology**
_Indian Institute of Science, Bengaluru | AUG 2015 - JUL 2020_
Masters thesis: “Analysis of the human gut microbiome by Nanopore-based metagenomic sequencing” (Grade: 8/8)
 + KVPY National Science Fellowship (All-India rank: 335)


Research experience
======

**PhD candidate**
_University of Lausanne | AUG 2020 - PRESENT_
+ Led honeybee gut microbiome evolution research using next-generation sequencing and custom-made [Snakemake pipelines](https://github.com/Aiswarya-prasad/honeybee-cross-species-metagenomics/).
+ Designed and conducted experiments on microbiota-free bees with synthetic bacterial communities to study processes influencing community assembly using PacBio long-read sequencing.
+ Collaborated with multi-disciplinary teams in Malaysia and Kenya and initiated a new collaboration in India, expanding the study’s scope with cross-country data and mitigating delays due to pandemic restrictions in Malaysia by over one year.


**Master thesis research**
_Indian Institute of Science, Bengaluru | MAY 2019 - MAY 2020_
+ Established Nanopore sequencing system in a low-resource setting at 10% of the cost of other commonly available systems.
+ Developed an end-to-end pipeline from sample collection to bioinformatic analysis for metagenomics; tested various approaches for sample preparation and metaproteomic analysis of fecal samples using LC-MS/MS.
+ Analyzed the human gut microbiome in chronic pancreatitis patients, studying disease-related changes in collaboration with doctors. 


**iGEM undergraduate research**
_Indian Institute of Science, Bengaluru | MAY 2016 - OCT 2016_
+ Co-founded the first team from the institute and secured a bronze medal at iGEM in Boston, a competition for innovative biotech solutions using genetically engineered machines.
+ Contributed to the designing and cloning of a quorum sensing-based auto-inducible system for protein overexpression in bacteria.
+ Engaged in teamwork to secure funding for research and travel via three grants.


Publications
======

Mazel, F., **Prasad, A.**, Engel, P. (2024) Host specificity of gut microbiota associated with social bees: patterns and processes. Microbiology and Molecular Biology Reviews (In Review)

**Prasad A.**, Pallujam A. D., _et al_ (2024). Symbiont loss and gain, rather than co-diversification shapes honeybee gut microbiota diversity and function. [bioRxiv](https://doi.org/10.1101/2024.09.11.612390), 2024.09.11.612390 (In Review)

Baud, G. L., **Prasad, A.**, Ellegaard, K. M., & Engel, P. (2023). Turnover of strain-level persity modulates functional traits in the honeybee gut microbiome between nurses and foragers. [Genome Biology](https://doi.org/10.1186/s13059-023-03131-4), 24(1), 283.

Sarton-Lohéac, G., Nunes da Silva, C. G., Mazel, F., Baud, G., de Bakker, V., Das, S., El Chazli, Y., Ellegaard, K., Carcia-Carcera, M., Glover, N., Liberti, J., Nacif Marçal, L., **Prasad, A.**, Sommerville, V., SAGE class 2019-2020 and 2020-2021, Bonilla-Rosso, G., & Engel, P. (2023). Deep divergence and genomic diversification of gut symbionts of neotropical stingless bees. [Mbio](https://doi.org/10.1128/mbio.03538-22), 14(2), e03538-22.

Beal, J., Haddock-Angelli, T., Baldwin, G., Gershater, M., Dwijayanti, A., Storch, M., ... & with the **iGEM Interlab Study Contributors**. (2018). Quantification of bacterial fluorescence using independent calibrants. PloS one, 13(6), e0199432.


Extracurricular Engagement
======

The Consulting Society Incubator – Senior member and Learning Unit Manager
_École Polytechnique Fédérale de Lausanne (EPFL) | MAY 2024 (Onwards)_

Workshop on Decolonizing Global North-South Research Collaboration
_Centre for Development and Environment, University of Bern (Online) | AUG 2022_

Moderator, Health & Medical Data Privacy Committee
_iGEM India BIOSUMMIT (Remote) | JUL 2020_

iGEM Delegate, United Nations Convention on Biological Diversity
_Sharm El-Sheikh, Egypt | NOV 2018_ -->
