---
layout: archive
permalink: /research/wild_vs_managed
author_profile: true
title: "Gut microbiome of wild and managed honeybees"
collection: research
venue: "University of Lausanne"
location: "Switzerland"
order: 3
start_date: 2024-08-01
end_date: 2025-08-01
# media:
#   - path: "/images/research/Honeybees_wild_managed.png"
#     alt: "Drawing of wild and managed honeybees"
#     focal_point: Smart
#     width: 450
#     height: 300
#   - path: "/images/research/Honeybees_Microbiome_drawing.png"
#     alt: "Drawing of honeybee gut microbiome research scheme"
#     focal_point: Smart
#     width: 450
#     height: 300
---

Little is known about the gut microbiome of wild honeybees. Taking advantage of a large collection of wild honeybees of various subspecies from across Africa, painstakingly put together by our collaborators, we are investigating the differences in the gut microbiome of wild and managed honeybees. We are using shotgun metagenomics to resolve strain-level differences in the gut bacterial community and are currently analyzing the data to identify bacterial strains and species that are found in wild and managed honeybees and investigating their distribution across colonies of different subspecies.

### Context

- In Europe and North America, nearly all honeybees are managed or feral populations derived from managed stock  
- Truly wild honeybee populations are now mostly restricted to parts of **Africa and Asia**, where they occur in native habitats and show rich **subspecies-level diversity**  
- With their **simple and specialized gut microbiomes**, honeybees serve as a powerful model to investigate how **domestication and management affect microbial evolution**

---

### Challenge

- **Assembling a global wild bee collection** is logistically complexâ€”thanks to an extensive international network of collaborators across Africa, and expert field ecologists in Germany, we now have hundreds of samples from genetically distinct wild colonies  
- The differences we seek are expected to be **subtle and strain-specific**, requiring **deep shotgun metagenomic sequencing** and **modern bioinformatics pipelines** to achieve the necessary resolution  

---

### Expectations and Preliminary Results

- We have identified **species-level bacterial patterns** shared between wild and managed bees  
- We observed lear patterns of **strain-level variation** across managed colonies and subspecies
- From our [previous work](https://www.biorxiv.org/content/10.1101/2024.09.11.612390v1.full) we expected that the strain-level differences in the gut microbiome will reflect the [evolutionary history](https://www.nature.com/articles/s41598-023-35937-4) of the honeybee subspecies. However, early results suggest that the differences are driven by whether the colonies are managed or wild, rather than by subspecies.

<!-- Little is known about the gut microbiome of wild honeybees. Taking advantage of a large collection of wild honeybees of various subspecies from across Africa, painstakingly put together by our collaborators, we are investigating the differences in the gut microbiome of wild and managed honeybees. We are using shotgun metagenomics to resolve strain-level differences in the gut bacterial community and are currently analyzing the data to identify bacterial strains and species that are found in wild and managed honeybees and investigating their distribution across colonies of different subspecies.

**Context**: 
  * In Europe or North America, honeybees (_Apis mellifera_) that you've seen flying about was most likely from a managed farm or escaped from one (this is being freshly debated). 
  * "Wild" honeybees are only found across parts of Africa and Asia which is their natural habitat but, they have been transported by humans across the world over the past few centuries. 
  * With their simple and specialized gut microbiomes, honeybees make a great model to study the effects of domestication on the gut microbiome.

**Challenge**:
  * _Locating and sampling wild honeybees_ from across the world is a daunting task. Thanks to our _international collaborators_ across Africa and amazing field ecologists from Germany, we have assembled an extensive collection of hundreds of honeybee samples.
  * The _differences are likely to be sublte_ and cannot be eaily detected by standard (amplicon sequencing) methods. We are using _high-throughput_ next-genetation sequencing to implement shotgun metagenomics and modern bioinformatic approaches to resolve strain level differences in the gut bacterial community.

**Expectations and Results**:
  * We have identified a number of species that are found in wild and managed honeybees and are currently investigating their distribution across colonies of different types.
  * Our collection of wild honeybees span across different contries in Africa with each location harboring a different subspecies of _Apis mellifera_. From our [previous work](https://www.biorxiv.org/content/10.1101/2024.09.11.612390v1.full) we expect that the strain-level differences in the gut microbiome will reflect the [evolutionary history](https://www.nature.com/articles/s41598-023-35937-4) of the honeybee subspecies. -->